documentation/puppetdb_connection.markdown

```diff
@@ -0,0 +1,85 @@
+---
+title: "PuppetDB 2.2 » Configuring PuppetDB Connection"
+layout: default
+canonical: "/puppetdb/latest/puppetdb_connection.html"
+---
+
+[puppetdb_root]: ./index.html
+[connect_to_puppetdb]: ./connect_puppet_master.html
+[confdir]: /puppet/latest/reference/dirs_confdir.html
+[puppetdb_conf]: ./connect_puppet_master.html#edit-puppetdb\.conf
+[url_prefix_setting]: ./configure.html#url-prefix
+[web_router_setting]: ./configure.html#:web-router-service
+
+The `puppetdb.conf` file contains the hostname and port of the [PuppetDB][puppetdb_root] server. It is only used if you are using PuppetDB and have [connected your Puppet master to it][connect_to_puppetdb].
+
+Summary
+-----
+
+The Puppet master makes HTTPS connections to PuppetDB to store catalogs, facts and new reports. It also uses PuppetDB to answer queries such those necessary to support exported resources. If the PuppetDB instance is down, depending on the configuration of the Puppet master, it could cause the Puppet run to fail. This document discusses configuration options for the `puppetdb.conf` file, including settings to make the PuppetDB terminus more tolerant of failures.
+
+## Location
+
+The `puppetdb.conf` file is always located at `$confdir/puppetdb.conf`. Its location is **not** configurable.
+
+The location of the `confdir` varies; it depends on the OS, Puppet distribution, and user account. [See the confdir documentation for details.][confdir]
+
+## Example
+
+    [main]
+    server_urls = https://puppetdb.example.com:8081/
+
+## Format
+
+The `puppetdb.conf` file uses the same INI-like format as `puppet.conf`, but only uses a `[main]` section
+
+`[main]` Settings
+-----
+
+The `[main]` section defines all of the PuppetDB terminus settings.
+
+### `server_urls`
+
+This parameter specifies how the Puppet master should connect to PuppetDB. The configuration should look something like the configuration below:
```

Comment:
Let's always say "setting" instead of "parameter," if we're talking about settings in a config file. In Days of Yore we called basically everything a parameter, and it made normal conversations difficult. :)

Also, s/like the configuration below:/like this example: